flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/WindowAggregateQueryOperation.java

```diff
@@ -225,5 +249,27 @@ public String asSummaryString() {
                     throw new IllegalStateException("Unknown window type: " + type);
             }
         }
+
+        public String asSerializableString(String table) {
+            switch (type) {
+                case SLIDE:
+                    return String.format(
+                            "HOP((%s), DESCRIPTOR(%s), %s, %s)",
+                            table,
```

Comment:
since `%s` can become large, should we also do some line break and indention here